src/pages/PhotoList.js

```diff
@@ -0,0 +1,118 @@
+import axios from 'axios';
+import { Component } from 'react';
+import styled from 'styled-components';
+import Masonry from 'react-masonry-css';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import Loader from 'react-loader-spinner';
+import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
+
+const breakpointColumns = {
+  default: 3,
+  1200: 3,
+  992: 3,
+  768: 2,
+  576: 1,
+};
+
+export default class PhotoList extends Component {
+  state = {
+    photos: [],
+    isLoading: false,
+    page: 1,
+  };
+
+  componentDidMount = () => {
+    this.fetchPhotos();
+  };
+
+  /**
+   * This method returns a list of photos from the api
+   */
+  fetchPhotos = async () => {
+    try {
+      // set isLoading to true
+      this.setState({ isLoading: true });
+
+      // make a fetch request to the api GET/photos end point
+      const response = await axios.get(
+        `${process.env.REACT_APP_UNSPLASH_API_URL}/photos?page=${this.state.page}&per_page=20`,
+        {
+          headers: {
+            Authorization: `Client-ID ${process.env.REACT_APP_UNSPLASH_ACCESS_KEY}`,
+          },
+        }
+      );
```

Comment:
something to think about: can this request be put in a function? 

https://floreacristian.teachable.com/courses/from-beginner-to-professional-developer1/lectures/31565669